README.md

```diff
@@ -8,7 +8,7 @@ Verify links in [awesome](status/status.md) projects
 [![Gem Version](https://badge.fury.io/rb/awesome_bot.svg)](https://badge.fury.io/rb/awesome_bot)
 [![](https://img.shields.io/badge/awesome-status-brightgreen.svg)](status/status.md)
 
-`awesome_bot` checks for valid URLs in a file, it can be used to [verify pull requests](#validate-pull-requests) updating a README.
+`awesome_bot` checks for valid URLs in a file. It can be used to [verify pull requests](#validate-pull-requests) which update a README.
```

Comment:
Not interested in this change, please undo

Reply:
FYI: https://en.wikipedia.org/wiki/Comma_splice

Reply:
Thanks, still not interested.